core/sr-io/without_std.rs

```diff
@@ -385,6 +385,20 @@ pub mod ext {
 		/// - nonzero otherwise.
 		fn ext_submit_transaction(data: *const u8, len: u32) -> u32;
 
+		/// Returns information about the local node's network state.
+		///
+		/// # Returns
+		///
+		/// - The encoded `Result<offchain::LocalNetworkState, ()>`.
+		fn ext_local_network_state(msg_len: *mut u32) -> *mut u8;
+
+		/// Returns the locally configured authority public key, if available.
+		///
+		/// # Returns
+		///
+		/// - The encoded `Result<PublicKey encoded to Vec<u8>, ()>`.
```

Comment:
- Can we have a description what does `crypto` mean here?
- The same comments as for `ext_local_network_state` regarding `msg_len` and semantics of the return value